src/components/Patient/symptoms/list.tsx

```
@@ -38,32 +39,40 @@ export function SymptomsList({
-  const { data: symptoms, isLoading } = useQuery({
-    queryKey: ["symptoms", patientId, encounterId],
-    queryFn: query(symptomApi.listSymptoms, {
-      pathParams: { patientId },
-      queryParams: {
-        encounter: encounterId,
-        ordering: "-created_date",
-      },
-    }),
+  const {
+    data: symptomsData,
+    isLoading: isLoading,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)